File: main/formulas/density.py
#d = m  / V
# V = volume
# m = massa
# d = densidade

def density(d=0,m=0,v=0):
    if v == 0 and (d and m) != 0:
        resultado = m / d
        return [d, m, resultado]
    elif m == 0 and (v and d) != 0:
        resultado = d * v
        return [d, resultado, v]
    elif d == 0 and (v and m) != 0:
        resultado = m / v
        return [resultado, m, v]
    elif d and m and v != 0:
        return [d, m, v]
    elif (d and v and m) == 0:
        return [d, m, v]    
    else:
        return [d, m, v]
        
    


if __name__ == "__main__":
    print(density())

File: deprecated/formulas/density.js
function density(d = 0, m = 0, v = 0) {
    var resultado;
  
    if (v === 0 && (d && m) !== 0) {
      resultado = m / d;
      return [d, m, resultado];
    } else {
      if (m === 0 && (v && d) !== 0) {
        resultado = d * v;
        return [d, resultado, v];
      } else {
        if (d === 0 && (v && m) !== 0) {
          resultado = m / v;
          return [resultado, m, v];
        } else {
          if (d && m && v !== 0) {
            return [d, m, v];
          } else {
            if ((d && v && m) === 0) {
              return [d, m, v];
            } else {
              return [d, m, v];
            }
          }
        }
      }
    }
  }